homeassistant/components/google_assistant/smart_home.py

```diff
@@ -93,8 +108,27 @@ def entity_to_device(entity: Entity):
         supported = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
         for feature, trait in class_data[2].items():
             if feature & supported > 0:
+                _LOGGER.debug(trait)
                 device['traits'].append(trait)
 
+                # Actions require this attributes for a device
+                # supporting temperature
+                # For IKEA trådfri, these attributes only seem to
+                # be set only if the device is on?
+                if entity.domain == light.DOMAIN:
```

Comment:
These don't need to be nested in the for loop, in fact that's less efficient. They can simply be inside the `if class_data[2]:`.

Reply:
colorModel is default RGB...you think we still need to set it?